blocks/core/ff_container/ff_container-expandable/ff_container-expandable.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+$(function(){ 
+	$("[data-ff-action='expandable']").click(function(){
+		var $expandableContent = $("[data-ff='expandable-content']"),
+		$expandableIcon = $("[data-icon='expandable-icon']"),
+		$expandableText = $("[data-ff='expandable-text']");
+		
+		var $expandableOpenText = $expandableText.attr("data-open-text"),
+		$expandableClosedText = $expandableText.attr("data-closed-text");
+
+		var isVisible = $expandableContent.is(":visible");
+		var newText = isVisible ? $expandableClosedText : $expandableOpenText;
+
+		if (isVisible) {
+			$expandableContent.hide();
+			$expandableIcon.removeClass("ff_icon-page-up-open-blue").addClass("ff_icon-page-down-open-blue");
```

Comment:
@mwilliamson-firefly The icons are generated from svgs using a build script, and the colours are generated too, as well as the corresponding css which inlines the svg. As it's a separate file, we can't really extend it in less. We can't really import into the theme file either as the page needs to figure out support and include the relevant file, or fallbacks if necessary. Will need to think more on this, as I've encountered it too.